client/README.md

```diff
@@ -4,21 +4,21 @@
 
 James Bond went on a new mission and this time as a [Secret Service provider](https://specifications.freedesktop.org/secret-service/latest/).
 
-The library consists of two modules:
+Similar to [libsecret](https://gitlab.gnome.org/GNOME/libsecret), this library allows to store secrets using two different backends:
```

Comment:
Explaining what the library does by comparing to another one that is specific to gnome doesn't make it clearer...

Reply:
How would you describe the relation to libsecret? This is one part that confused me, so I think it is important to explain this clearly. For example, it wasn't clear to me that both libsecret and oo7 provide both a secret service and a file backend.

Reply:
I agree here, defining the new thing in terms of the old thing is not ideal. For example:

oo7-client is a library intended to be used by apps (or anything that could need to store/retrieve secret really (clients)) that allows you to
 
- Interact with a encrypted local storage for secrets (sandbox-friendly way) using the Secrets portal (https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Secret.html). 
- Interact with the secret service API (insert link to https://specifications.freedesktop.org/secret-service/latest/)

Of course the difficult part is writing this succinctly.

> How would you describe the relation to libsecret

oo7-client and libsecret are libraries for clients of both secret APIs
oo7-portal and gnome-keyring are servers of the portal API
oo7-(yet-to-be-named https://github.com/bilelmoussaoui/oo7/pull/73) and gnome-keyring are servers of the fdo secret API

> provide both a secret service backend and a file backend.

The FDO api is a *system* api, it is not intended to be used by apps, anyone interacting with it can see every secret. For apps you want the file backend and for system components you want the FDO api.

Reply:
I removed the reference to libsecret from the top and instead extended the "How does it compare to other libraries?" section. I wonder whether the that section should be moved up because it is easy to miss after the extensive examples. But examples are important, too. What do you think?